js/src/product-feed/product-statistics/status-box.js

```diff
@@ -85,14 +85,12 @@ function getSyncResult( {
 	};
 }
 
-const issuesQuery = {
-	page: 1,
-	per_page: ISSUE_TABLE_PER_PAGE,
-};
-
 function FeedStatus() {
-	const { data } = useAppSelectDispatch( 'getMCIssues', issuesQuery );
-	const status = getUnsolvedStatusText( data?.total );
+	const issues = useMCIssues();
+	const status = getUnsolvedStatusText(
+		issues[ ISSUE_TYPE_ACCOUNT ]?.data?.total +
+			issues[ ISSUE_TYPE_PRODUCT ]?.data?.total
```

Comment:
💅 
What about moving this to `useMCIssues` as a return property such as `mcTotal`, and then it could be also used in the `<IssuesTableCard>` at:
https://github.com/woocommerce/google-listings-and-ads/blob/4a36c202dc326a69e0d44c64107190afbf2ea2f5/js/src/product-feed/issues-table-card/index.js#L57-L61

Reply:
Added the total prop here -> https://github.com/woocommerce/google-listings-and-ads/pull/1176/commits/4e9d7f090bc3fb0930173f962698c2bb879997be

I updated `status-box` as well. 

Regarding `<IssuesTableCard>` I'm fine doing that there as well... but to keep in mind that then we will need to wait until both, products and account and other potential issues to complete the resolution in order to decide if we need to render. With this approach we show the Issues table to the user as soon as we get issues in one of both.  

Thoughts?

Reply:
> Regarding `<IssuesTableCard>` I'm fine doing that there as well... but to keep in mind that then we will need to wait until both, products and account and other potential issues to complete the resolution in order to decide if we need to render. With this approach we show the Issues table to the user as soon as we get issues in one of both.
> 
> Thoughts?

This also makes sense. I don't usually do this specifically to show UI a tens or a few hundred milliseconds earlier, but it's thoughtful to do so. 👍  So I think it's good to keep the purpose.